codec-http2/src/main/java/io/netty/handler/codec/http2/Http2DataWriter.java

```diff
@@ -32,7 +32,8 @@
      * @param ctx the context to use for writing.
      * @param streamId the stream for which to send the frame.
      * @param data the payload of the frame. This will be released by this method.
-     * @param padding the amount of padding to be added to the end of the frame
+     * @param padding additional bytes that should be added to obscure the true content size. Must be between 0 and
+     *                256 (inclusive).
```

Comment:
explain how this padding value will be translated to bytes on the wire. it may be unexpected that 256 is allowed when the frame definition only allows for 8 byte length.